kimchi/src/circuits/lookup/index.rs

```diff
@@ -119,6 +119,9 @@ impl<T> LookupSelectors<T> {
             chacha_final,
             lookup_gate,
         } = self;
+        // This closure isn't really redundant -- it shields the parameter from a copy -- but
```

Comment:
ah I see, but I'm not sure I'm more clever than clippy here because I don't get how this is true

Reply:
Try deleting it :man_shrugging: